python/tk_desktop/project_model.py

```diff
@@ -245,8 +244,7 @@ def __init__(self, parent, overlay_parent_widget):
         """ Constructor """
         ShotgunModel.__init__(self, parent, download_thumbs=True)
 
-        # override the default connection to use one that is tied to the current login
-        connection = ShotgunLogin.get_instance_for_namespace("tk-desktop").get_connection()
+        connection = sgtk.platform.current_engine().sgtk.shotgun
```

Comment:
The engine has a `shotgun` property that you can use! - e.g. `sgtk.platform.current_engine().shotgun`